icu4c/source/test/intltest/usettest.cpp

```diff
@@ -1811,6 +1823,195 @@ void UnicodeSetTest::TestSymbolTable() {
             logln(UnicodeString("Ok, got ") + us.toPattern(a, true));
         }
     }
+    for (const auto &[variables, expression, expectedErrorCode, expectedPattern] :
+         std::vector<std::tuple<std::vector<std::pair<std::u16string_view, std::u16string_view>>,
+                                std::u16string_view, UErrorCode, std::u16string_view>>{
```

Comment:
Trying to make this a bit more readable...
Consider a named struct with a constructor??
```suggestion
    for (const auto &[variables, expression, expectedErrorCode, expectedPattern] :
            std::vector<std::tuple<
                std::vector<std::pair<std::u16string_view, std::u16string_view>>,
                std::u16string_view,
                UErrorCode,
                std::u16string_view>>{
```